tests/monitor/test_data_monitoring.py

```diff
@@ -211,3 +244,5 @@ def test_data_monitoring_send_alerts(snowflake_data_monitoring_with_alerts_in_db
     assert len(alerts) == len(expected_alerts)
     assert alerts[0].id == expected_alerts[0].id
     assert type(alerts[0]) == type(expected_alerts[0])
+
+
```

Comment:
Great job Aaron, thanks so much for working on this and looking forward to merge it back to master! :)

Reply:
Thank you very much Or! Let me know if there is anything else I can do to help you all out.